tests/test_base.py

```diff
@@ -608,6 +609,25 @@ def test_it_enforces_allowed_schemes(self, value, expected):
             with pytest.raises(ValidationError, match=expected_err_msg):
                 field.clean(value)
 
+    @pytest.mark.parametrize(
+        'value, expected',
+        [
+            ('https://example.com/', 'https://example.com/'),
```

Comment:
Maybe also test that a non-default, not-explicity-disallowed scheme works as well (e.g. `ftp://`)?